app/resources/views/components/eyes.blade.php

```diff
@@ -0,0 +1,71 @@
+<div class="eyes">
+    <svg
+        xmlns="http://www.w3.org/2000/svg"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        viewBox="0 0 300 300"
+        width="150px"
+        height="150px"
+    >
+        <defs>
+            <path id="star" d="M21 29L8.7 38l4.7-14.5-12.4-9h15.3L21 0l4.7 14.6H41l-12.4 8.9L33.3 38z" fill="#FFCC00" transform="translate(-20 -20)" />
+
+            <ellipse id="white" cx="30" cy="45" rx="30" ry="45" fill="#fff" filter="url(#inset-shadow)" />
+
+            <symbol id="pupil">
+                <g class="pupil" x-bind:transform="`translate(${translate.x}px ${translate.y}px`">
```

Comment:
Die runde Klammer wird glaube ich nicht geschlossen?

Lernfrage: Wieso braucht es die backticks und die Anführungzeichen?

Reply:
```suggestion
                <g class="pupil" x-bind:transform="`translate(${translate.x}px ${translate.y}px)`">
```

Reply:
Und noch eine Lernfrage: Woher kommt das translate Objekt?

Reply:
> Lernfrage: Wieso braucht es die backticks und die Anführungzeichen?

Das ist einfach eine Template-Expression

Reply:
> Und noch eine Lernfrage: Woher kommt das translate Objekt?

Hier wird der entsprechende Getter definiert:

https://github.com/HowTheyVote/epvotes/pull/524/files#diff-7e49ffb499db4db084d7aac8714ff690b3f74198fce07427fd7877564d542902R21

Und wir können darauf zugreifen, weil wir das als [globalen Data-Kontext](https://alpinejs.dev/globals/alpine-data) definiert haben. (Das ist aber einfach nur eine Möglichkeit, das in eine separate JS-Datei auszulagern, ansonsten ist es nichts besonderes.)

Reply:
Ah, da hab ich nicht mitgedacht, das ergibt natürlich Sinn. :+1: